BangazonTerminalInterface/Controllers/PaymentController.cs

```diff
@@ -8,5 +9,31 @@ namespace BangazonTerminalInterface.Controllers
 {
     class PaymentController
     {
+        string paymentType; //To Capture the payment type
+        int paymentAccountNumber; //To capture the account number
+        string newPayment; //To build the new payment that will be added to the DB
+        
+        private void requestPaymentType() // Ask for payment Type
```

Comment:
@debragordon A small suggestion but I would suggest trying to use WriteToConsole() inside of the CustomerController.cs for the Console.WriteLine. This is the function I wrote(with Dustins help) to Print out generic info to the console and return the users response. Other than that everything looks good in your PaymentController.